tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -14,7 +14,22 @@
 //
 //******************************************************************************
 
-#include "DrawingTest.h"
+#import "DrawingTest.h"
+#import <windows.h>
+#import "CGContextInternal.h"
+
+static woc::unique_cf<CGColorSpaceRef> rgbColorSpace(CGColorSpaceCreateDeviceRGB());
```

Comment:
Yep, this is gone in the next iteration. 

---
In reply to: [88732063](https://github.com/Microsoft/WinObjC/pull/1414#discussion_r88732063) [](ancestors = 88732063)